staging/src/k8s.io/apimachinery/pkg/runtime/serializer/cbor/internal/modes/decode_test.go

```diff
@@ -288,16 +575,32 @@ func TestDecode(t *testing.T) {
 			}
 
 			t.Run(fmt.Sprintf("mode=%s/%s", modeName, tc.name), func(t *testing.T) {
+				if tc.failureReason != "" {
+					t.Skip(tc.failureReason)
```

Comment:
Please add a comment here to make it clear that our intention is to fix all known failure cases and remove the skip.

Reply:
Done.